metabase/metabase
# fix bars width when chart has two y-axes

### Description

Bar widths are not computed correctly when charts have two y-axes.

### How to verify

- Create a bar chart with >=3 series and two y-axes
- Ensure bar widths are the same as they would be on a single y-axis chart

### Demo

One y-axis:

<img width="1363" alt="Screenshot 2024-05-23 at 8 53 11 PM" src="https://github.com/metabase/metabase/assets/14301985/8add74f3-3032-4fdf-84a9-a14a485772f8">

Two y-axes:

<img width="1370" alt="Screenshot 2024-05-23 at 8 52 27 PM" src="https://github.com/metabase/metabase/assets/14301985/4461c1d2-463b-4152-9ab4-d9a910ef5043">

Two y-axes before:

<img width="1355" alt="Screenshot 2024-05-23 at 9 00 44 PM" src="https://github.com/metabase/metabase/assets/14301985/fb206872-e6e9-4999-b0ac-53b57a6be679">

### Checklist
